tests/statement_serializer_tests/statements/remove.cpp

```diff
@@ -11,19 +11,20 @@ TEST_CASE("statement_serializer remove") {
     };
     auto table = make_table("users", make_column("id", &User::id, primary_key()), make_column("name", &User::name));
     using storage_impl_t = internal::storage_impl<decltype(table)>;
-    auto storageImpl = storage_impl_t{table};
+    storage_impl_t storageImpl{table};
     using context_t = internal::serializer_context<storage_impl_t>;
     context_t context{storageImpl};
-    std::string value;
-    decltype(value) expected;
+
+    std::string value, expected;
```

Comment:
please avoid declaring more than one var within one statement - it is a very rare technique and it makes code less recognizable by most of developers

Reply:
OK.
TBH I don't find it hard to read at all. It's even easier on the brain than all the auto and decltype(auto) variables. Developers know it from for-loops or from javascript, and all the template code is so complex that this simple thing doesn't really add any complexity.
Just stating my viewpoint.